homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -73,7 +73,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 # How long to wait for a state change to happen
-STATE_CHANGE_WAIT_TIMEOUT = 2.0
+STATE_CHANGE_WAIT_TIMEOUT = 5.0
```

Comment:
I can make some local test with Alexa to see how long it will wait for before aborting and giving an error.

Then we can make the timeout value just slightly below that.